src/styles/main.scss

```diff
@@ -0,0 +1,1590 @@
+@import "vendor-port.css";
+@import "font";
+
+/*variable file*/
+@import "variables";
+
+
+
+/*ng cloak default styling*/
+
```

Comment:
How is this file different from `base.scss`? Doesn't `base.scss` serve a similar purpose?
@Sanji515 may also help in the discussion here.

Reply:
Yes! just copy the releavant `.scss` that is required now to the `base.scss`.